main.tf

```diff
@@ -52,7 +52,7 @@ resource "aws_db_instance" "postgresql" {
 #
 
 resource "aws_cloudwatch_metric_alarm" "database_cpu" {
-  alarm_name          = "alarm${var.environment}DatabaseServerCPUUtilization"
+  alarm_name          = "alarm-${var.environment}-${var.database_identifier}_DatabaseServerCPUUtilization"
```

Comment:
Would it be possible to reorganize this (and the others) so that they look like:

```
alarm${var.environment}DatabaseServerCPUUtilization-${var.database_identifier}
```

A nitpick, but we are trying to keep the camelcase thing consistent. The identifier at the end breaks that, but at least in a way that is somewhat consistent with CloudFormation behavior.